packages/upload/src/vaadin-upload-mixin.js

```diff
@@ -608,6 +608,15 @@ export const UploadMixin = (superClass) =>
         }
       }
 
+      const containsFolders = Array.from(dropEvent.dataTransfer.items)
```

Comment:
Let's maybe leave a comment why we need a conditional here.

Reply:
Added a comment.